code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -159,9 +163,7 @@
 						reagents.trans_to(target, reagents.total_volume)
 					bitecount++
 					On_Consume(target)
-			return 1

```

Comment:
If you don't know why did you remove it, it prevents the next proc called in the chain of attack thus it should block afterattack from being called.